components/supervisor/frontend/src/ide/ide-metrics-service-client.ts

```diff
@@ -8,7 +8,9 @@ import { serverUrl } from '../shared/urls';
 
 export enum MetricsName {
     SupervisorFrontendClientTotal = "gitpod_supervisor_frontend_client_total",
-    SupervisorFrontendErrorTotal = "gitpod_supervisor_frontend_error_total"
+    SupervisorFrontendErrorTotal = "gitpod_supervisor_frontend_error_total",
+    SupervisorFrontendSessionErrored = "gitpod_supervisor_frontend_sessions_errored",
```

Comment:
I think we need something like `gitpod_supervisor_frontend_unique_error_total` which reports a single once per page for (resource, error) group. See internal thread https://gitpod.slack.com/archives/C01KGM9BH54/p1662533595773799?thread_ts=1662496101.039309&cid=C01KGM9BH54